### PR TITLE
Improve provisioned host workflow

### PR DESCRIPTION
**Pull request by lzap**
We do not provide description of production host boot sequence.

### CODE REVIEW

File: guides/common/modules/con_provisioning-workflow.adoc [hunk 1]
```diff
@@ -16,6 +16,7 @@ When you complete all the options for the new host, submit the new host request.
 * PXELinux, Grub, Grub2, and iPXE configuration files for the host in the TFTP {SmartProxyServer} that is associated with the subnet.
 * A Puppet certificate on the associated Puppet server.
 * A realm on the associated identity server.
+. The host is configured to boot from network as the first device and HDD as the second device.
```
Comment (melcorr) [inline guides/common/modules/con_provisioning-workflow.adoc:19]:
```suggestion
. The host is configured to boot from the network as the first device and HDD as the second device.
```

File: guides/common/modules/con_provisioning-workflow.adoc [hunk 2]
```diff
@@ -48,6 +49,20 @@ endif::[]
 . The boot loader initiates boot from the local drive.
 . If you configured the host to use any Puppet classes, the host configures itself using the modules.
 
+The fully provisioned host performs the following workflow:
+
+. The host is configured to boot from network as the first device and HDD as the second device.
```
Comment (melcorr) [inline guides/common/modules/con_provisioning-workflow.adoc:54]:
```suggestion
. The host is configured to boot from the network as the first device and HDD as the second device.
```

File: guides/common/modules/con_provisioning-workflow.adoc [hunk 3]
```diff
@@ -48,6 +49,20 @@ endif::[]
...
+. The new host requests a DHCP reservation from the DHCP server.
+. The DHCP server responds to the reservation request and returns TFTP `next-server` and `filename` options.
+. The host requests the boot loader and menu from the TFTP server according to the PXELoader setting.
+. A boot loader is returned over TFTP.
+. The boot loader fetches configuration for the host through its provisioning interface MAC address.
```
Comment (melcorr) [inline guides/common/modules/con_provisioning-workflow.adoc:59]:
```suggestion
. The boot loader fetches the configuration settings for the host through its provisioning interface MAC address.
```

**Review by melcorr — CHANGES_REQUESTED** [review]
Reads pretty well. 
Few small changes.

**Comment by lzap** [comment]
My God those articles are killing me. Appended a commit please squash on merge, TY!

**Pull request merged**